src/unittests/maths.spec.ts

```diff
@@ -3,20 +3,19 @@ Util math unit tests.
 `;
 
 import { makeTestGroup } from '../common/framework/test_group.js';
-import { kBit } from '../webgpu/shader/execution/builtin/builtin.js';
+import { anyOf, CaseList, deepEquals, kBit } from '../webgpu/shader/execution/builtin/builtin.js';
```

Comment:
It doesn't seem right that we have `src/unittests` test depending on `../webgpu/shader/execution/builtin`

Reply:
It's a little odd, but I think unittests/ (which runs on presubmit) is the right place for testing helper code, and webgpu/ is the right place for the helper code. We have several other webgpu/ helpers tested in unittests/ so it's at least consistent.

We do enforce limitations on imports between different parts of the source tree: `.eslintrc.json` has a few specific lints set up. If we wanted to change it, we could add rules there.

Reply:
I agree, there is an issue with this. I have redone this code to isolate the builtin specific stuff to that file and only implement a generic cartesian pair functions in maths.spec.ts.